CHANGELOG.md

```diff
@@ -3,9 +3,89 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 4.0.0
+
+_Aug 23, 2021_
+
+This is the first offitial stable release of the `DataGrid` and `DataGridPro`. It supports `@material-ui/core` v4 and has a partial support for v5. From this release on we will release new features for the `DataGrid` and `DataGridPro` under version `5.0.0-beta.0` and have full support for `@material-ui/core` v5.
+
+Big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Introduce the [row editing](https://material-ui.com/components/data-grid/editing/#row-editing) feature (#2098) @m4theushw
```

Comment:
I think that we could add a quick screen recording to explain what the feature is about (for the lazy developers). I could do it if we find the proposal valuable.